util/regexp-assemble/regexp-942370.data

```diff
@@ -0,0 +1,26 @@
+[\"'`]\s*?\*.+or\W*?[\"'`]\d
+[\"'`]\s*?\*.+xor\W*?[\"'`]\d
+[\"'`]\s*?\*.+div\W*?[\"'`]\d
+[\"'`]\s*?\*.+like\W*?[\"'`]\d
+[\"'`]\s*?\*.+between\W*?[\"'`]\d
+[\"'`]\s*?\*.+and\W*?[\"'`]\d
+[\"'`]\s*?\*.+id\W*?[\"'`]\d
+\^[\"'`]
+^[\w\s\"'`-]+(?<=and\s)(?<=or)(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+^[\w\s\"'`-]+(?<=and\s)xor(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+^[\w\s\"'`-]+(?<=and\s)div(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+^[\w\s\"'`-]+(?<=and\s)like(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+^[\w\s\"'`-]+(?<=and\s)between(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+^[\w\s\"'`-]+(?<=and\s)and\s(?<=xor\s)(?<=nand\s)(?<=not\s)(?<=\|\|)(?<=\&\&)\w+\(
+[\"'`][\s\d]*?[^\w\s]+\W*?\d\W*?.*?[\"'`\d]
```

Comment:
@franbuehler i removed this part from 942370 and created PL3 rule 942470 for it

Reply:
Thank you @emphazer.
I didn't touch rule 942370 because of your open PR and because I haven't fully tested it yet.
I wait until your PR is merged. I will extend my PR afterwards. Ok?

Reply:
@franbuehler 
thanks. lets wait until today evening.